presentation/src/main/java/com/anytypeio/anytype/presentation/sets/GalleryViewMapper.kt

```diff
@@ -173,24 +173,31 @@ private suspend fun getCoverFromRelationOrLayout(
     if (coverRelationKey != null) {
         val relation = storeOfRelations.getByKey(coverRelationKey)
         coverImage = relation?.let {
-            getCoverImageFromRelation(it, obj, coverRelationKey, store, urlBuilder, isLargeSize)
+            getCoverImageFromRelation(
+                relation = it,
+                obj = obj,
+                coverRelationKey = coverRelationKey,
+                dependedObjects = dependedObjects,
+                urlBuilder = urlBuilder,
+                isLargeSize = isLargeSize
+            )
         }
     }
     if (coverImage == null && obj.layout == ObjectType.Layout.IMAGE) {
         coverImage = if (isLargeSize) {
             urlBuilder.large(obj.id)
         } else {
-            urlBuilder.large(obj.id)
+            urlBuilder.medium(obj.id)
```

Comment:
can it affect the image resolution in editor ?

Reply:
no